superduper/misc/special_dicts.py

```diff
@@ -418,6 +432,8 @@ def _component_info(obj):
                     value = str(value)[:50] + "..."
                 else:
                     value = str(value)
+            elif isinstance(value, list):
```

Comment:
lets add test for tuple as well

`elif isinstance(value, (tuple, list))`

Reply:
Agreed.